tests/averaging/translation/test_translation_averaging_base.py

```diff
@@ -26,6 +30,65 @@ def setUp(self):
 
         self.obj = DummyTranslationAveraging()
 
+    def __execute_test(
+        self, i2Ui1_input: Dict[Tuple[int, int], Unit3], wRi_input: List[Rot3], wti_expected: List[Point3]
+    ) -> None:
+        """Helper function to run the averagaing and assert w/ expected.
+
+        Args:
+            i2Ri1_input (Dict[Tuple[int, int], Rot3]): [description]
+            wRi_expected (List[Rot3]): [description]
+        """
+        if isinstance(self.obj, DummyTranslationAveraging):
+            self.skipTest("Test case invalid for dummy class")
+
+        wti_computed = self.obj.run(len(wRi_input), i2Ui1_input, wRi_input)
+
+        wTi_computed = [Pose3(wRi, wti) for wRi, wti in zip(wRi_input, wti_computed)]
+        wTi_expected = [Pose3(wRi, wti) for wRi, wti in zip(wRi_input, wti_expected)]
+        self.assertTrue(
+            geometry_comparisons.compare_global_poses(
+                wTi_computed, wTi_expected, RELATIVE_ERROR_THRESHOLD, ABSOLUTE_ERROR_THRESHOLD
+            )
+        )
+
+    def test_circle_two_edges(self):
```

Comment:
Does this pass? It is also under constrained.

Reply:
Oh yeah it is underconstrained. But what if I go from 4 points on the circle to 6?

Reply:
Oh yeah it will still be underconstrained. But the test passes somehow.

Reply:
I think someone fixing the gauge between the 0th and 1st input makes this problem adequately constrained?